tests/utils/test_utils.go

```diff
@@ -77,3 +78,12 @@ func RunL2CommandAndParseJSON(cmdArgs ...string) (map[string]EnvData, error) {
 
 	return envMap, nil
 }
+
+func GetAbsolutePath(relPath string) (string, error) {
+	// GetAbsolutePath takes a relative path and returns its absolute path.
+	absPath, err := filepath.Abs(relPath)
+	if err != nil {
```

Comment:
I don't see the benefit of this function. We could simply call `filepath.Abs` directly wherever needed (since that also returns two values)

Reply:
Right